CesiumGS/cesium
# Fix error when kml descriptions have tags with missing url attribtues.

Hi team,

This PR fixes a small issue where  Cesium throws an unexpected error when trying to load a kml/kmz file containing html tags with a missing url attribute. Example: KML with `script`/`img` tag with missing `src` attribute.  

This fix will correctly handle the `null` values returned from `element.getAttribute()` calls.

### Testing
Before: 
  - Open this public [sandcastle link](https://sandcastle.cesium.com/#c=bVDLTsMwEPwVK6cgUVscgbQClVtbcQji5IvjbFurfkTedUv79dgNQuGxt92dmZ1ZHTwSOxo4QWRz5uHEloAmOf5+ndWy0td+GTwp4yHK6uZRen3lHZx9UaTakKKGTP+irqZjboPqa+kZk9WeaMAHIaI68Z2hfeoSQsxaBJ64Dk54dd8rMZ4UW/MxyydmziAav5th1KIdQKMo8mK1WQs0brDQ6mgG2oywNupnomi6RMAP7iIr6YvlMSTvv50hV31f/8gwwV1CcG/h97q6rRqks4VFSVTqKTsIkViKtuZcEGRDigBFl/QBcirEolqgjZhSm94cmenn/3yYaasQ82abrG3NBWS1aETG/6GW5+bMr0eIVp0LbH+3WI9Dznkjcvs/k0KwnYoT5RHxCQ) that loads a KMZ file
  - Observe that data is not rendered on the map 
  - The console also shows the errors
 
After:
 - Open this local [sandcastle link](http://localhost:8080/Apps/Sandcastle/index.html#c=bVDLTsMwEPwVK6cgUVscgbQClVtbcQji5IvjbFurfkTedUv79dgNQuGxt92dmZ1ZHTwSOxo4QWRz5uHEloAmOf5+ndWy0td+GTwp4yHK6uZRen3lHZx9UaTakKKGTP+irqZjboPqa+kZk9WeaMAHIaI68Z2hfeoSQsxaBJ64Dk54dd8rMZ4UW/MxyydmziAav5th1KIdQKMo8mK1WQs0brDQ6mgG2oywNupnomi6RMAP7iIr6YvlMSTvv50hV31f/8gwwV1CcG/h97q6rRqks4VFSVTqKTsIkViKtuZcEGRDigBFl/QBcirEolqgjZhSm94cmenn/3yYaasQ82abrG3NBWS1aETG/6GW5+bMr0eIVp0LbH+3WI9Dznkjcvs/k0KwnYoT5RHxCQ) for this branch
 - Observe that the data is rendered correctly without errors

Thanks in advance. 
Please let me know if I need to make any changes to the PR.

Cheers,
Nanda
